docs/README.md

```diff
@@ -11,6 +11,7 @@
 - [`<Button />`](Button.md)
 - [`<Link />`](Link.md)
 - [`<Hug />`](Hug.md)
+- [`<Flag />`](Flag.md)
```

Comment:
Have you added this to the main README.md as well?

Reply:
Missed that, but updated